test/check-application

```diff
@@ -391,5 +397,32 @@ class TestNavigator(testlib.MachineCase):
         b.click("div.pf-c-modal-box__close")
         m.execute("sudo chattr -i /home/admin/newdir2")
 
+    def testHiddenItems(self):
+        b = self.browser
+        m = self.machine
+
+        self.restore_dir("/home/admin")
```

Comment:
This is redundant https://github.com/cockpit-project/cockpit/blob/main/test/common/testlib.py#L1499